Carmunication/Controllers/LoginViewController.swift

```diff
@@ -10,29 +10,81 @@ import CryptoKit
 import FirebaseAuth
 import UIKit
 
-// 애플 로그인 파이어베이스 인증 시 재전송 공격을 방지하기 위해 요청에 포함시키는 임의의 문자열 값
-fileprivate var currentNonce: String?
-
 class LoginViewController: UIViewController {
+    // 애플 로그인 파이어베이스 인증 시 재전송 공격을 방지하기 위해 요청에 포함시키는 임의의 문자열 값
+    fileprivate var currentNonce: String?
```

Comment:
currentNonce 프로퍼티는 클래스 내부에서만 쓰이는 것 같은데 private가 더 적절하지 않을까요?
```suggestion
    private var currentNonce: String?
```

Swift 접근제어자에 대해 확인해보시는 것도 좋을 것 같아요!

Reply:
파이어베이스 가이드를 보고 그대로 작성한 코드였는데 어차피 해당 변수 호출하는 함수가 다 같은 클래스에 있어서 private이 적절할 것 같네요!